static/cordova.js

```diff
@@ -34,4 +35,15 @@
         pickContact: function () {}
     };
 
+    // https://github.com/floatinghotpot/cordova-plugin-sms/tree/master/docs
+    window.SMS = window.SMS || {
+        sendSMS: function(addresses, message, onSuccess, onErr) {
+            // Shortcut for testing errors
+            if (addresses[0] === '123 123 123') {
+                return onErr(new Error('We can\'t seem to contact 123 123 123'));
+            }
+            onSuccess('your sms was sent');
```

Comment:
We should probably discuss this whole idea of having a non-functional mock here but yeah that's what it's for, for now.